README.md

```diff
@@ -43,31 +43,34 @@ a lot of data, and may rely on unreleased versions of CIAO).
 
 ## How to create a tarball for the CIAO CM team
 
+This has **changed** in CIAO 4.14, bith the internals of the script
```

Comment:
`with`

Reply:
For reference, when you click a line you get this bizarre "+ in a box" icon 

![sherpa](https://user-images.githubusercontent.com/224638/140102770-1c882c07-00e9-42a7-ab55-dfb6241ba8ef.png)


if you click it (or press control-g I think, it will create something like

    ```suggestion
    This has **changed** in CIAO 4.14, bith the internals of the script
    ```

and if you edit the text with your suggestion it can actually be added automatically by the reader. e.g.

```suggestion
This has **changed** in CIAO 4.14, with the internals of the script
```